x-pack/plugins/secops/public/apps/start_app.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { createHashHistory } from 'history';
+import React from 'react';
+import { ApolloProvider } from 'react-apollo';
+import { ThemeProvider } from 'styled-components';
+
+// TODO use theme provided from parentApp when kibana supports it
+import { EuiErrorBoundary } from '@elastic/eui';
+import * as euiVars from '@elastic/eui/dist/eui_theme_k6_light.json';
+import { AppFrontendLibs } from '../lib/lib';
+import { PageRouter } from '../routes';
+
+export async function startApp(libs: AppFrontendLibs) {
```

Comment:
Looking at the mixture of named functions and ES2015+ pure non-named functions/arrow function expression I think that maybe some of this mixed style was because for a while people couldn't use the `async` keyword together with `export` but we should be good to do this now:

```
export const startApp = async (libs: InfraFrontendLibs) => {
```

To keep our styles even more consistent with ES2015 arrow expressions. Testing on my side and it looks good. Unless I'm missing a specific reason to be mixing them, I prefer to go with the 2015 arrow function to reduce the iky `this` issues.